src/material/chips/chip-listbox.ts

```diff
@@ -210,14 +209,12 @@ export class MatChipListbox
   override _chips: QueryList<MatChipOption> = undefined!;
 
   ngAfterContentInit() {
-    if (this._pendingInitialValue !== undefined) {
-      Promise.resolve().then(() => {
-        this._setSelectionByValue(this._pendingInitialValue, false);
-        this._pendingInitialValue = undefined;
-      });
-    }
-
     this._chips.changes.pipe(startWith(null), takeUntil(this._destroyed)).subscribe(() => {
+      if (this.value !== undefined) {
```

Comment:
I think this should be `if (this.value === undefined) {`. Previously we were checking `_pendingInitialValue !== undefined`, which implied the value _was_ `undefined`, as the pending value had not been set. Hopefully making this change will resolve the test failures I'm seeing in google

Reply:
This change `if (this.value === undefined)` fails even more unit tests

Reply:
@mmalerba How can I know whether my changes fixed the problem? Where can I look at it?

Reply:
I will have to rerun the internal test and report back

Reply:
@mmalerba  Any updates? I have synced  the branch

Reply:
It's looking better - there's still 3 failing tests, 1 ExpressionChangedAfterChecked and 2 screenshot diffs that look possibly wrong. It's not clear if these are issues with the changes or issues with the tests themselves. Unfortunately I don't think there's much we can do until I have some time to dig deeper into it

Reply:
Thanks for letting me know! In case you need me to update the branch just ping